apps/web/app/(app)/simple/SimpleList.tsx

```diff
@@ -47,28 +47,33 @@ import { HtmlEmail } from "@/components/email-list/EmailPanel";
 import { ViewMoreButton } from "@/app/(app)/simple/ViewMoreButton";
 
 export function SimpleList(props: {
-  messages: ParsedMessage[];
+  initialMessages: ParsedMessage[];
   nextPageToken?: string | null;
   userEmail: string;
   type: string;
 }) {
   const { toHandleLater, onSetHandled, onSetToHandleLater } =
     useSimpleProgress();
 
+  const [messages, setMessages] = useState(props.initialMessages);
   const router = useRouter();
 
   const [parent] = useAutoAnimate();
 
   const [isPending, startTransition] = useTransition();
 
-  const toArchive = props.messages
+  const toArchive = messages
```

Comment:
We still want to archive unsubscribed messages. This would skip that.

This should work better. But don't use this filter in `toArchive`.

```ts
const [unsubscribed, ...] = useState()
```